server/src/tenant/schema-builder/graphql-types/scalars/date-time.scalar.ts

```diff
@@ -1,8 +1,8 @@
 import { GraphQLScalarType } from 'graphql';
 import { Kind } from 'graphql/language';
 
-export const DatetimeScalarType = new GraphQLScalarType({
-  name: 'Datetime',
+export const DateTimeScalarType = new GraphQLScalarType({
+  name: 'DateTime',
```

Comment:
While we are at it, can we update generate-object-type.util to not use String for createdAt and updatedAt? 😄